File: Java/Logic-1/nearTen.java

public class logic1 {

	// given a non-negative number "num", return true if num is within 2 of a multiple of 10
	public boolean nearTen(int num) {
		
		boolean b = false;
		
		if (num >= 0)
			if (num % 10 <= 2 || num % 10 >= 8)
				b = true;
			
		return b;
		
	}
	
	public static void main(String[] args) {
		
		logic1 o = new logic1();
		
		System.out.println(o.nearTen(12));
		System.out.println(o.nearTen(17));
		System.out.println(o.nearTen(19));

	}

}

File: Python/Logic-1/near_ten.py

# given a non-negative number "num", return true if num is within 2 of a multiple of 10
def near_ten(num):
    b = False
    
    if num >= 0:
        if num % 10 <= 2 or num % 10 >= 8:
            b = True
    
    return b

print(near_ten(12))
print(near_ten(17))
print(near_ten(19))
